doc/source/conf.py

```diff
@@ -64,6 +65,7 @@
             f"ansys-sphinx-theme-v{convert_version_to_pymeilisearch(__version__)}": "ansys-sphinx-theme",  # noqa: E501
         },
     },
+    "announcement": "https://raw.githubusercontent.com/ansys/ansys-sphinx-theme/main/doc/_templates/message.html",  # noqa: E501
```

Comment:
Use a path object in here so we do not depend on a hard-coded link.

Reply:
Added. Thanks @jorgepiloto